engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropNode.java

```diff
@@ -61,11 +73,44 @@
     private static final int RADIUS = 1024;
 
     private WorldRenderer worldRenderer;
+    private BackdropProvider backdropProvider;
+    private RenderingConfig renderingConfig;
+    private WorldProvider worldProvider;
 
     private int skySphere = -1;
     private SetWireframe wireframeStateChange;
 
+    private Material skyMaterial;
+
+    @SuppressWarnings("FieldCanBeLocal")
+    @Range(min = 1.0f, max = 8192.0f)
+    private float sunExponent = 512.0f;
+    @SuppressWarnings("FieldCanBeLocal")
+    @Range(min = 1.0f, max = 8192.0f)
+    private float moonExponent = 256.0f;
+    @SuppressWarnings("FieldCanBeLocal")
+    @Range(min = 0.0f, max = 10.0f)
+    private float skyDaylightBrightness = 0.6f;
+    @SuppressWarnings("FieldCanBeLocal")
+    @Range(min = 0.0f, max = 10.0f)
+    private float skyNightBrightness = 1.0f;
+
+    @SuppressWarnings("FieldCanBeLocal")
+    private Vector3f sunDirection;
+    @SuppressWarnings("FieldCanBeLocal")
+    private Vector3d zenithColor;
+    @SuppressWarnings("FieldCanBeLocal")
+    private SubmersibleCamera activeCamera;
+    @SuppressWarnings("FieldCanBeLocal")
+    private Vector3f cameraDir;
+    @SuppressWarnings("FieldCanBeLocal")
+    private Vector3f cameraPosition;
+
     public BackdropNode(Context context) {
+        backdropProvider = context.get(BackdropProvider.class);
+        renderingConfig = context.get(Config.class).getRendering();
+        worldProvider = context.get(WorldProvider.class);
+
         worldRenderer = context.get(WorldRenderer.class);
         Camera playerCamera = worldRenderer.getActiveCamera();
```

Comment:
As it turns out, we don't.

That got eliminated in the cleanup stage of this Node.